onnxruntime/test/providers/cpu/tensor/split_op_test.cc

```diff
@@ -677,6 +706,9 @@ TEST(SplitOperatorTest, Split18_NumOutputs_EvenSplit) {
                       7.f, 8.f}});
 
   int64_t num_outputs = 2;
+#ifdef USE_COREML
```

Comment:
there are a few more tests within `#ifdef USE_COREML`. could those tests be run for other EPs too or is it necessary to only enable them for the CoreML EP?

Reply:
those tests are for the option `is_initializer`, coreml requires input to be initializer. The default is set to true, but specifically for opset18 tests, marked as false. so figured might be necessary leave them run as false in this case.

but sure about the reason.

Reply:
if they can run for other EPs, we could also get additional test coverage more generally. and I suppose this may be handy for future NNAPI EP support.

Reply:
make sense. I am doing a test run currently with this option enabled for other eps now.

Reply:
check-in this pr for now. and will reconsider loosing enabling that for more general testing for other eps when doing nnapi split pr.